proposals/interface-types/working-notes/scenarios/cc.md

```diff
@@ -0,0 +1,251 @@
+# Pay by Credit Card
```

Comment:
It'd be nice to update this example to be in sync with record.md above.  E.g., I think `create` should be `pack`?

Reply:
agreed